mix.exs

```diff
@@ -2,45 +2,58 @@ defmodule Algae.Mixfile do
   use Mix.Project
 
   def project do
-    [app:     :algae,
-     name:    "Algae",
-
-     description: "Bootstrapped algebraic data types for Elixir",
-     version: "1.2.3",
-     elixir:  "~> 1.7",
-
-     package: [
-       maintainers: ["Brooklyn Zelenka"],
-       licenses:    ["MIT"],
-       links:       %{"GitHub" => "https://github.com/expede/algae"}
-     ],
-
-     source_url:   "https://github.com/expede/algae",
-     homepage_url: "https://github.com/expede/algae",
-
-     aliases: [
-       quality: ["test", "credo --strict"]
-     ],
-
-     deps: [
-       {:credo,    "~> 1.0",  only: [:dev, :test]},
+    [
+      app: :algae,
+      aliases: aliases(),
+      deps: deps(),
+
+      # Versions
+      version: "1.2.4",
+      elixir: "~> 1.9",
+
+      # Docs
+      name: "Algae",
+      docs: docs(),
+
+      # Hex
+      description: "Bootstrapped algebraic data types for Elixir",
+      package: package()
+    ]
+  end
 
-       {:dialyxir, "~> 0.5",  only: :dev},
-       {:earmark,  "~> 1.2",  only: :dev},
-       {:ex_doc,   "~> 0.16", only: :dev},
+  defp aliases do
+    [
+      quality: ["test", "credo --strict"]
+    ]
+  end
 
-       {:inch_ex,  "~> 2.0",  only: [:dev, :docs, :test]},
+  defp deps do
+    [
+      {:credo, "~> 1.1", only: [:dev, :test]},
+      {:dialyxir, "~> 0.5", only: :dev},
+      {:earmark, "~> 1.4.0", only: :dev},
+      {:ex_doc, "~> 0.21.2", only: :dev},
+      {:inch_ex, "~> 2.0", only: [:dev, :docs, :test]},
+      {:quark, "~> 2.3"},
+      {:type_class, "~> 1.2"},
+      {:witchcraft, "~> 1.0"}
+    ]
+  end
 
-       {:quark,      "~> 2.3"},
-       {:type_class, "~> 1.2"},
-       {:witchcraft, "~> 1.0"}
-     ],
+  defp docs do
+    [
+      extras: ["README.md"],
+      logo: "./brand/mini-logo.png",
+      main: "readme",
+      source_url: "https://github.com/witchcrafters/algae"
+    ]
+  end
 
-     docs: [
-       extras: ["README.md"],
-       logo: "./brand/mini-logo.png",
-       main: "readme"
-     ]
+  defp package do
+    [
+      licenses: ["MIT"],
+      links: %{"GitHub" => "https://github.com/witchcrafters/algae"},
+      maintainers: ["Brooklyn Zelenka"]
```

Comment:
@icidasset you should add yourself to `maintainers`

Also FYI I've created a Wicthcrafters Hex org. Sent you an invit 😉

Reply:
Thanks! ☺️ Will do both.

Reply:
@expede I just tried to publish `type_class` with the the `organization` thing and it's complaining about a missing subscription. I think that this `organization` property is for private packages only, no? 🤔 https://hex.pm/docs/private#publishing-a-private-package